.eslintrc.json

```diff
@@ -0,0 +1,271 @@
+{
+    "env": {
+        "es6": true
```

Comment:
Why enable es6? I think ES5 should be enforced which also means `parserOptions.ecmaVersion` needs to be set to `5`.